### ISSUE TITLE
ABCG2 / ABCB1 relative expression

### ISSUE THREAD

**Issue by Yuri05** [Open-Systems-Pharmacology/Dapagliflozin-Model#5]
- ABCG2 has a relative expression in blood cells of 0.00536585365853657 when coming from the DB, but is 0 in the Dapagliflozin-Project.
- ABCB1 in Dapagliflozin project has different expression than that of P-gp in other projects

_Originally posted by @PavelBal in https://github.com/Open-Systems-Pharmacology/OSP-PBPK-Model-Library/issues/62#issuecomment-1062884350_

**Comment by sfrechen** [comment]
@PavelBal @Yuri05 
Dapa is neither substrate of BCRP nor of P-gp. Thus, the expression does not have any relevance. The model was built using PK-Sim 8. There was no blood cell expression for efflux transporter. That may explain why it is 0 now. For P-gp: this is some historical remnant. The expression represents the original RT-PCR profile which was adjusted by Hanke et al 2018.

The topic may be revisited once we define our expression building block(s). For the current model, there is no action point.

By the way - I do also see now in PK-Sim v10 P-gp expression in blood cells... where do these number come from?

**Comment by Yuri05** [comment]
>By the way - I do also see now in PK-Sim v10 P-gp expression in blood cells... where do these number come from?

In v9 there were expressions in blood cells as well. Was the same database source used during the import?
![grafik](https://user-images.githubusercontent.com/25061876/157664927-2a9a243d-9ced-4969-9614-5bd14cd6faa6.png)
![grafik](https://user-images.githubusercontent.com/25061876/157665098-00ec97f5-6c39-4cb9-8f5e-9e7b01c4013f.png)

**Comment by sfrechen** [comment]
No blood cells shown in PK-Sim v8:
![image](https://user-images.githubusercontent.com/26569320/157712627-b794d6f6-f5e0-4573-b67a-b85e24ff827d.png)

**Comment by PavelBal** [comment]
Yes, I think PK-Sim did not allow transporters between blood cells and plasma.

**Comment by Yuri05** [comment]
>Yes, I think PK-Sim did not allow transporters between blood cells and plasma.

True. So the values were there, but hidden because unused.
